### DISCUSSION TITLE
Something went wrong setting up Supabase

### DISCUSSION THREAD

**Discussion by yuzhongrong** [supabase/supabase#32455] [Questions]
![WX20250101-015224@2x](https://github.com/user-attachments/assets/f7345c9e-4415-4b51-8d9b-119660176474)

**Comment by gregnr** [comment]
Hey @yuzhongrong, are you still experiencing this issue? I just tried creating a fresh Bolt app that links to Supabase and was not able to reproduce this error.

Some things that can help us debug this:
- Original prompt/messages sent to Bolt
- Confirming what steps you did to link your Supabase project (ie. clicking "Connect to Supabase", etc)
- How long after linking your Supabase project did that error appear?
